drivers/gles3/shader_gles3.h

```diff
@@ -398,9 +400,15 @@ class ShaderGLES3 {
 	void set_custom_shader(uint32_t p_code_id);
 	void free_custom_shader(uint32_t p_code_id);
 
-	uint32_t get_version() const { return new_conditional_version.version; }
-	bool is_version_ubershader() const { return (new_conditional_version.version & VersionKey::UBERSHADER_FLAG); }
-	_FORCE_INLINE_ bool is_version_valid() const { return version && version->compile_status == Version::COMPILE_STATUS_OK; }
+	uint32_t get_version() const {
+		return new_conditional_version.version;
+	}
+	bool is_version_ubershader() const {
+		return (new_conditional_version.version & VersionKey::UBERSHADER_FLAG);
+	}
+	_FORCE_INLINE_ bool is_version_valid() const {
+		return version && version->compile_status == Version::COMPILE_STATUS_OK;
+	}
```

Comment:
This (and `get_active` above) were wrongly changed due to a bug in clang-format 14.0.0: https://github.com/llvm/llvm-project/issues/54901

But it's not a big deal and it might actually be more readable like this for those.